src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -10,13 +10,16 @@
 import java.util.concurrent.TimeUnit;
 
 public class VideoFanTakeover {
```

Comment:
Does it make sense to extract common interface `VideoFanTakeover` and create a dedicated class for Fandom (and removing work `Fandom` from methods)? Or make separate class for Fandom only?